cli/cook/http.py

```diff
@@ -12,18 +12,24 @@
 timeouts = None
 
 
-def configure(config):
+
+def configure(config, static_modules):
     """Configures HTTP timeouts and retries to be used"""
     global session
     global timeouts
     http_config = config.get('http')
     modules_config = http_config.get('modules')
-    session_module_name = modules_config.get('session-module')
-    adapters_module_name = modules_config.get('adapters-module')
-    session_module = importlib.import_module(session_module_name)
-    adapters_module = importlib.import_module(adapters_module_name)
-    logging.getLogger(session_module_name).setLevel(logging.DEBUG)
```

Comment:
Dropped this line since the `requests` library doesn't actually have its own logger. The `requests` documentation says that you should configure the `urllib3` logger if you want debug output:
https://2.python-requests.org/en/master/api/#api-changes